docs/notebooks/unfinished/example_measure_elzerman_readout.py

```diff
@@ -0,0 +1,54 @@
+from qtt.measurements.acquisition import UHFLIScopeReader, UHFLIStimulus
```

Comment:
Rename the file to something else, the example is about triggered readout of the device, but not related to Elzerman.

Reply:
Maybe we can add the Elzerman related stuff next week and turn it into a full sample? We could then also convert the sample into a notebook.